code/game/objects/structures/mirror.dm

```diff
@@ -128,10 +131,11 @@
 		return
 
 	var/mob/living/carbon/human/H = user
+	var/userloc = H.loc

```

Comment:
why is this needed

Reply:
so he can remember the players initial position